web-console/future-openshift/pinning/pinning.md

```diff
@@ -0,0 +1,26 @@
+# Pinning
+
+## Projects list
+![1](img/projects1.png)
+* Users may set their favorite projects by clicking the star toggle at the beginning of a row.
+* The project will not change positions in the list when the star is toggled on or off.
+
+![2](img/projects2.png)
+* Users may view only their favorite projects by checking the "Show only favorites" checkbox next to the textbox filter.
+
+### Project selector
+![3](img/dropdown.png)
+* Project favorites will be reflected in the project dropdown, but cannot be edited there.
+* Dropdown sections and a horizontal separator will be used to visually distinguish favorite projects and all other projects.
+
+
+## Roles list
+![4](img/roles1.png)
+* The roles list will load with several roles automatically favorited. These roles are commonly used and pre-favorited to bring them to the attention of the user.
```

Comment:
This along with the project favoriting are great additions! It's nice to see in the project selection just how helpful marking a number of these as favorites can be. Is there a similar example to use when selecting a Role and how having a number of favorites bubble to the top?

All of this looks awesome! Nice work, @bmignano